cores/esp32/esp32-hal-gpio.c

```diff
@@ -204,7 +215,7 @@ extern void __attachInterruptFunctionalArg(uint8_t pin, voidFuncPtrArg userFunc,
     	interrupt_initialized = (err == ESP_OK) || (err == ESP_ERR_INVALID_STATE);
     }
     if(!interrupt_initialized) {
-    	log_e("IO %i ISR Service Failed To Start", pin);
```

Comment:
Please don't simplify the error messages. This will make it more difficult to debug.